Source/gg2/Scripts/GameServer/GameServerCreate.gml

```diff
@@ -131,6 +131,12 @@
             game_end();
             exit;
         }
+        if (string_length(pluginList) > 65535)
+        {
+            show_message("Error: you are requiring too many server-sent plugins.");

```

Comment:
Obviously a very, very unlikely scenario. But, well, I was burned by the lack of such a check for 255 bytes.